### DISCUSSION TITLE
Release Proposal for version 2.6.0

### DISCUSSION THREAD

**Discussion by PragmaTwice** [apache/kvrocks#1767] [General]
*Refer to https://kvrocks.apache.org/community/create-a-release.*

Now I propose to release Apache Kvrocks version `2.6.0` (a **feature release**). It has been over two months since the last feature release (`2.5.0`), and many exciting new features and fixes have been merged into the `unstable` branch.

All commits on unstable until 3092081a4857d681173a752a15b5b5b95e68758a (included) will be considered as commits for this release, but XGROUP related commands will be disabled to avoid confusing users (this is because the implementation of XGROUP is currently far from complete).

The deadline for the cherry-picking period has been set as **September 22, 2023, at 20:00 UTC+8**. No new commits will be accepted after this time.

Feel free to leave your comments about this release : )

cc @apache/kvrocks-committers 

---

Commits cherry-picked to `2.6`:
- #1763 
- #1769 
- #1771

**Comment by mapleFU** [comment]
+1 (binding)

Note:
1. We've upgrade RocksDB, but I think it's ok because it just contain some bugfixes ( https://github.com/apache/kvrocks/pull/1725 )
2. We've set the default arg for max-io-mb, this need to be in document https://github.com/apache/kvrocks/pull/1713
3. Though bloom is supported, but I think we didn't test bloom with slot-migration, it's still limited

**Reply by mapleFU** [reply]
Also, we can upgrade rocksdb to latest after release as https://github.com/apache/kvrocks/pull/1725 talks :-)

**Comment by torwig** [comment]
+1
As @mapleFU mentioned, the bloom filter won't be affected by slot migration (and I guess replication as well, but I'm not sure).

**Comment by git-hulk** [comment]
+1

> Though bloom is supported, but I think we didn't test bloom with slot-migration, it's still limited

We can mention this limitation in our release logs.

**Comment by caipengbo** [comment]
+1

> We can mention this limitation in our release logs.

I agree. Bloom can be labeled **experimental**

**Comment by PragmaTwice** [comment]
Hi everyone, the deadline has been reached.

I will prepare an ASF vote soon.
